butterknife/build.gradle

```diff
@@ -0,0 +1,21 @@
+apply plugin: 'com.android.library'
+
+android {
+  compileSdkVersion rootProject.ext.compileSdkVersion
+  buildToolsVersion rootProject.ext.buildToolsVersion
+
+  defaultConfig {
+    minSdkVersion rootProject.ext.minSdkVersion
+    targetSdkVersion rootProject.ext.targetSdkVersion
```

Comment:
Libraries should not have `targetSdkVersion` specified or you create manifest merging conflicts.